tensorflow/python/kernel_tests/BUILD

```diff
@@ -1665,18 +1683,36 @@ cuda_py_test(
     xla_enable_strict_auto_jit = True,
 )
 
-cuda_py_test(
-    name = "bias_op_test",
-    size = "medium",
-    srcs = ["bias_op_test.py"],
-    additional_deps = [
-        "//third_party/py/numpy",
+py_library(
+    name = "bias_op_base",
+    srcs = ["bias_op_base.py"],
+    deps = [
         "//tensorflow/python:array_ops",
         "//tensorflow/python:client_testlib",
         "//tensorflow/python:framework_for_generated_wrappers",
         "//tensorflow/python:gradients",
         "//tensorflow/python:nn_grad",
         "//tensorflow/python:nn_ops",
+        "//third_party/py/numpy",
+    ],
+)
+
+cuda_py_test(
+    name = "bias_op_deterministic_test",
+    size = "small",
```

Comment:
This and the bias_op_test test below are timing out, please change the size to 'medium'.